test/e2e/charts/flow-visibility/provisioning/tls/settings.xml

```diff
@@ -0,0 +1,16 @@
+<yandex>
+    <https_port>8443</https_port>
+    <tcp_port_secure>9440</tcp_port_secure>
+    <openSSL>
+        <server>
+            <certificateFile>/opt/certs/tls.crt</certificateFile>
+            <privateKeyFile>/opt/certs/tls.key</privateKeyFile>
+            <dhParamsFile>/etc/clickhouse-server/config.d/dhparam.pem</dhParamsFile>
```

Comment:
Given that providing `dhParamsFile` is optional, I think we should omit it for testing to keep things simpler

I actually don't even see `dhParamsFile` mentioned in https://clickhouse.com/docs/en/guides/sre/configuring-ssl. Modern ciphers should not require this. In any case, we should not really be worried about security hardening when it comes to e2e testing.

Reply:
Removed. Thanks.